boa3/model/builtin/method/strbytestringmethod.py

```diff
@@ -0,0 +1,22 @@
+import ast
+from typing import Dict, List, Tuple
+
+from boa3.model.builtin.method.strmethod import StrMethod
+from boa3.model.variable import Variable
+from boa3.neo.vm.opcode.Opcode import Opcode
+
+
+class StrByteStringMethod(StrMethod):
+
+    def __init__(self):
+        from boa3.model.type.type import Type
+        args: Dict[str, Variable] = {
+            'object': Variable(Type.union.build([Type.str, Type.bytes])),
+        }
```

Comment:
you can use ByteString here instead of Union[str, bytes]
```suggestion
        from boa3.model.type.primitive.bytestringtype import ByteStringType
        from boa3.model.type.type import Type
        args: Dict[str, Variable] = {
            'object': Variable(ByteStringType.build()),
        }
```